config/examples/paperclip.yml

```diff
@@ -11,3 +11,6 @@ development:
 
 test:
   <<: *<%= ENV['FILE_UPLOAD_STORAGE'] || 'filesystem' %>
+
+production:
+  <<: *<%= ENV['FILE_UPLOAD_STORAGE'].presence || 'filesystem' %>
```

Comment:
OCD alarm!

Reply:
wdym?

Reply:
@akostadinov It's just that the empty line is missing at the end of the file :upside_down_face: